services/core-api/app/api/mines/permits/permit_amendment/resources/permit_amendment.py

```diff
@@ -177,6 +183,26 @@ def post(self, mine_guid, permit_guid, permit_amendment_guid=None):
             )
             new_pa.related_documents.append(new_pa_doc)
 
+        if now_application_guid is not None and permit_amendment_status_code == "DFT":
+            application_identity = NOWApplicationIdentity.find_by_guid(now_application_guid)
+            if application_identity.now_application:
+                now_type = application_identity.now_application.notice_of_work_type_code
+                condition_code = now_type
+
+                if now_type == "QCA" or now_type == "QIM":
```

Comment:
Can change to
```
                if now_type == "QIM":
                    condition_code = "QCA"
                elif now_type == "COL":
                    condition_code = "MIN"
```

Reply:
Can we actually make this logic in the `StandardPermitConditions.find_by_notice_of_work_type_code` function? That way the caller doesn't need to modify the value based on some logic beforehand.

Reply:
moved the logic!